pkg/deploy/generator/scripts/gatewayVMSS.sh

```diff
@@ -434,17 +434,6 @@ cat >/etc/default/vsa-nodescan-agent.config <<EOF
   }
 EOF
 
-# we start a cron job to run every hour to ensure the said directory is accessible
-# by the correct user as it gets created by root and may cause a race condition
-# where root owns the dir instead of syslog
-# TODO: https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/12591207
-cat >/etc/cron.d/mdsd-chown-workaround <<EOF
-SHELL=/bin/bash
-PATH=/bin
-0 * * * * root chown syslog:syslog /var/opt/microsoft/linuxmonagent/eh/EventNotice/arorplogs*
-EOF
-
-echo "enabling aro services"
```

Comment:
I feel like we'll still want these `echo` messages for the loop that comes after